pkg/gomega/assertions/eventually_configmap.go

```diff
@@ -0,0 +1,27 @@
+package assertions
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/onsi/gomega"
+	"github.com/openshift/osde2e-common/pkg/clients/openshift"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// EventuallyConfigMap is a gomega async assertion that can be used with the
+// standard or custom gomega matchers
+// Polls the resource every 30 seconds until success or timeout defined by POLLING_TIMEOUT
+//
+//	EventuallyConfigMap(ctx, client, configMapName, namespace).Should(BeAvailable(), "config map %s should exist", configMapName)
```

Comment:
The BeAvailable matcher only works on `Deployment` objects, we will need to work out a `Exists()` matcher or something similar. `.ShouldNot(BeNil())` may be sufficient